packages/trees/frontend/sirius-components-trees/src/treeitems/TreeItemContextMenu.tsx

```diff
@@ -132,7 +134,22 @@ export const TreeItemContextMenu = ({
           const element = React.createElement(component, props);
           return element;
         })}
-
+        {item.hasChildren ? (
+          <MenuItem
+            key="expand-all"
+            data-testid="expand-all"
+            onClick={() => {
+              onExpandAll(item);
+              onClose();
+            }}
+            disabled={readOnly}
+            aria-disabled>
+            <ListItemIcon>
+              <ExpandMore fontSize="small" />
```

Comment:
I know that that icon is supposed to represent the same kind of action, and this is subjective, but visually it's "just" a downward facing ">", and to me it does not really convey the notion of "recursive expansion" / "show all".

Reply:
Ok, I changed for the "Unfold More" icon.